src/i18n/locales/sl.json

```diff
@@ -0,0 +1,105 @@
+{
+  "forms-validation-error-required": "To polje je obvezno.",
+  "message-login-failed": "Prijava ni bila uspešna. Prosimo preverite vaše geslo.",
+  "message-server-unreachable": "Strežnik ni dostopen: Preverite internetno povezavo in URL strežnika. Preverite tudi, ali vaš Opencast sistem dovoljuje dostop prek Studio-ja..",
+  "message-upload-complete": "Nalaganje zaključeno!",
+  "message-upload-complete-explanation": "Vaš posnetek bo sedaj sprocesiran v Opencast sistemu.",
+  "cancel-button-title": "Prekini",
+  "pause-button-title": "Začasno ustavi snemanje",
+  "record-button-title": "Začni snemanje",
+  "resume-button-title": "Obnovi snemanje",
+  "nav-open-menu-button": "Menu",
+  "nav-recording": "Snemanje",
+  "nav-settings": "Nastavitve",
+  "nav-about": "O nas",
+  "review-button-discard-and-record": "Prekinite in ponovno posnemite",
+  "save-creation-button-upload": "Naloži v Opencast",
+  "save-creation-form-invalid": "Prosimo določi naslov in avtorja ",
+  "save-creation-label-presenter": "Avtor",
+  "save-creation-label-title": "Naslov",
+  "save-creation-title": "Skoraj zaključeno!",
+  "save-creation-title-done": "Zaključeno!",
+  "save-creation-subsection-title-download": "Shrani posnetek",
+  "save-creation-download-button": "Shrani",
+  "save-creation-subsection-title-upload": "Naloži na Opencast",
+  "save-creation-upload-to": "Naloži na <1>{{server}}</1>",
+  "save-creation-new-recording": "Začni novo snemanje",
+  "save-creation-new-recording-warning": "Z pričetkom novega snemanja bodo trenutni posnetki izgubljeni (to ne vpliva na že naložene ali prenesene posnetke). Ali ste prepričan?",
+  "share-desktop": "Delite zaslon",
+  "share-camera": "Delite video kamero",
+  "state-paused": "Zaustavljeno",
+  "state-recording": "Snemanje",
+  "state-waiting": "Čakanje",
+  "settings-header": "Nastavitve",
+  "settings-language": "Jezik",
+
+  "back-button-label": "Nazaj",
+  "next-button-label": "Naprej",
+
+
+  "confirm-discard-recordings": "Ali ste prepričani, da želite zavreči posnetke? Tega ni mogoče razveljaviti!",
+  "confirm-cancel-recording": "Trenutno snemate! Ali ste prepričani, da želite preklicati in s tem zavreči snemanje?",
+
+  "sources-video-display-selected": "Zaslon je izbran",
+  "sources-video-display-and-user-selected": "Zaslon & video kamera izbrana",
+  "sources-video-user-selected": "Video kamera je izbrana",
+  "sources-video-question": "Izberite željeni video vir za snemanje",
+  "sources-video-none-available": "Vaš brskalnik / naprava ne podpira zajema zaslona ali video kamere:-(",
```

Comment:
I think there is missing a space before ":-(". Or does Slovenian not put a space there?